cmd/satellite/main.go

```diff
@@ -1,28 +1,43 @@
 package main
 
 import (
-	"encoding/json"
-	"fmt"
-	"github.com/gpuctl/gpuctl/internal/status/handlers"
+	"log/slog"
 	"time"
+
+	"github.com/gpuctl/gpuctl/internal/femto"
+	"github.com/gpuctl/gpuctl/internal/uplink"
 )
 
 func main() {
-	// Just get the GPU data and print it in JSON
-	gpuHandler := handlers.NvidiaGPUHandler{}
-	for {
-		res, err := gpuHandler.GetGPUStatus()
-		if err != nil {
-			fmt.Printf("Failed to parse: %v\n", err)
-			return
-		}
 
-		ser, err := json.Marshal(res)
+	log := slog.Default()
+
+	s := satellite{
+		// TODO: Make this configurable
+		gsAddr: "http://localhost:8080",
```

Comment:
I'd suggest that this reads from a configuration file for now, until we have DNS-based service discovery that's automated. Maybe we need to build in a way to parameterise our satellites through a TOML file?

Reply:
I'd rather do this with a command line arg at the start, but yeah.

Reply:
For long-running (and typically daemonised) processes, it's industry norm to parameterise it with a configuration file rather than command line arguments especially if the program's configuration needs to be easily persisted through restarts. Complex configuration, and hot-loading configurations, maintainability and config modification also definitely benefit from this plumbing in place.

Worth taking some time to discuss this group-wise?